### DISCUSSION TITLE
503 Temporarily Unavailable for Spaceship Authentication via `pilot`, starting a few hours ago (Oct 18)

### DISCUSSION THREAD

**Discussion by gemmakbarlow** [fastlane/fastlane#26367] [Q&A] [answered]
Seeing 503 Temporarily Unavailable with our previously operational Fastlane setup, on both `2.223.1` and `2.224.0` as of a few hours ago. Is anyone else experiencing the same ?

`--verbose` shows me 

![image](https://github.com/user-attachments/assets/22948ce8-5694-4269-9aff-72fc51dc04ca)

**Comment by davidmdt21** [comment]
hello , yes same for me ... i think this problem has about apple api server or fastlane server /??? any body know ?
Please let me know if you find out about this

**Comment by gemmakbarlow** [comment, marked as answer]
Update: changing to use API keys for authentication (rather than the old session-based approach) appears to have resolved the issue for us. 

I recommend this article if you're looking for details on how to do that - https://www.runway.team/blog/how-to-build-the-perfect-fastlane-pipeline-for-ios

**Comment by kpowelltech** [comment]
This is an issue for us as well.  However, switching to use API keys for authentication is impossible at this juncture as I manage over a thousand apps and I don't have  API keys for them.  Fastlane documentation states that they still support the session-based approach.

**Reply by MattKiazyk** [reply]
Apple is returning 503 on the `https://idmsa.apple.com/appleauth/auth/signin` call that Fastlane, Xcodes and other session based login's use. Hopefully Apple fixes whatever they broke, but more likely the tools will have to find another way around if we can.
